tests/unit/client.py

```diff
@@ -262,28 +268,69 @@ def avoid_network_race_condition():
     for timeout in range(10):
         que = db.session.query(NetworkingAction).count()
         if (que > 0):
-            time.sleep(1)
+            time.sleep(.5)
```

Comment:
Since this is done a number of times, and the client tests increased the runtime by 300%, I figured reducing sleep time where possible can help.

Increased the range() to meet your concern.